Dockerfile

```diff
@@ -91,32 +102,50 @@ RUN pnpm install -g node-version-audit \
 
 # Set GOBIN to /usr/local/bin for Go binaries
 ENV GOBIN=/usr/local/bin
+ENV PATH="${GOBIN}:${PATH}"
 
 # Install gitxray
 RUN . $HOME/.asdf/asdf.sh \
     go install github.com/kulkansecurity/gitxray@latest
 
 # Install git-secrets
-RUN git clone https://github.com/awslabs/git-secrets.git $HOME/secrets \
-    && cd $HOME/secrets \
+RUN git clone https://github.com/awslabs/git-secrets.git git-secrets
+RUN cd git-secrets \
     && sudo make install \
-    && rm -rf $HOME/secrets
+    && rm -rf secrets
 
 # Install detect-secrets
 RUN pipx install detect-secrets
 
-# Install pmapper
-RUN pipx install pmapper
+# Install gitleaks
+RUN git clone https://github.com/gitleaks/gitleaks.git gitleaks \
+    && cd gitleaks \
+    && make build
+
+# Install gitxray
+RUN pipx install gitxray
 
 # Install gh-fake-analyzer
-RUN cd $HOME \
-    && git clone https://github.com/shortdoom/gh-fake-analyzer.git \
-    && cd gh-fake-analyzer \
+RUN git clone https://github.com/shortdoom/gh-fake-analyzer.git
```

Comment:
Is there any reason to split this into multiple `RUN` statements? I'd prefer to keep a low count of `RUN`s as it reduces layer count and maintains atomicity

Reply:
I'll send a new PR merging all RUNs individually for each tool